tests/unit/test_query_model.py

```diff
@@ -308,6 +309,20 @@ def test_infer_types_where_possible_even_without_schema():
         Function.EQ(event_count, Value("some_string"))
 
 
+def test_combination_of_typed_and_untyped_series():
+    # We're checking here that the type-checking can correctly combine typed series (`IsNull` has type `bool`) and
+    # untyped ones (`SelectColumn` has type `Any` because no schema is provided).
+    assert Function.EQ(
+        lhs=Function.IsNull(Value(None)),
+        rhs=SelectColumn(SelectPatientTable("p0"), "b1"),
+    )
```

Comment:
What is the `assert` for?

Reply:
I do this too, just because it feels weird to have a test with no asserts when all I really want to check is that no exception is thrown. I tell myself it's good to check that the function I'm calling has at least returned _something_.

Reply:
I do this because Dave does it. (Literally -- I looked around in the codebase for something similar just for uniformity's sake.) I think I'd mildly prefer a comment like `# all's well if we got here`, or something, but not enough to want to change it.

Reply:
Haha! And there I was feeling cheered in my idiosyncrasies because "Ben does it too".